ebusd-2.1.x/de/vaillant/15.350.csv

```diff
@@ -44,16 +44,19 @@ r;wi,,OffsetDesTemp,,,,,"2300",,,temp,,,
 r;wi,,StateOfRoomCon,,,,,"2400",,,UCH,,,
 r;wi,,LegioProtectionState,,,,,"2500",,,onoff,,,
 r;wi,,StatusDcf,,,,,"2600",,,dcfstate,,,
-r;wi,,DcfDaten,,,,,"2700",,,UIN,,,
+[SW<114]r;wi,,DcfDaten,,,,,"2700",,,UIN,,,
+[SW>=114]r;wi,,DesiredTempDay,,,,,"2700",,,D1C,,,
 r;wi,,IsInHoliday,,,,,"2800",,,UCH,,,
 r;wi,,IsInParty,,,,,"2900",,,onoff,,,
 r;wi,,IsInSingleHwcLoadingMode,,,,,"2A00",,,onoff,,,
 r;wi,,RestoreOpModeAfterHoliday,,,,,"2B00",,,onoff,,,
 r;wi,,IsInSavingsFunction,,,,,"2C00",,,onoff,,,
 r;wi,,SavingsFunctionTime,,,,,"2D00",,,time;onoff,,,
 r;wi,,SwitchOffParty,,,,,"2E00",,,onoff,,,
-r;wi,,IsInTeleSwitch,,,,,"2F00",,,onoff,,,
-r;wi,,IsInFloorPavingDrying,,,,,"3000",,,onoff,,,
+[SW<114]r;wi,,IsInTeleSwitch,,,,,"2F00",,,onoff,,,
+[SW>=114]r;wi,,Mode,,,,,"2F00",,,BCD,,, # Day=3 Low=4 Auto=2 Summer=5.
```

Comment:
is this key/value list right? if so, use the divisor/values column to assign them

Reply:
Yes, this key value list is right.

I've now reworked this pull request and used the contents of 15.370.csv for 15.150.csv. It was brought to my attention by @wlcrs that [others ](https://fromeijn.nl/connected-vaillant-to-home-assistant/) successfully use the 15.370.csv for their VRT350.